datashader/bundling.py

```diff
@@ -33,9 +33,9 @@ def distance_between(a, b):
     return (((a[0] - b[0]) ** 2) + ((a[1] - b[1]) ** 2))**(0.5)
 
 
-@nb.jit
+@nb.jit(forceobj=True)
 def resample_segment(segments, new_segments, min_segment_length, max_segment_length, ndims):
```

Comment:
This function uses numpy array creation and can therefore not be compiled in nopython mode. Maybe could be rewritten?

Reply:
The real problem here is that looplifting is also disabled due to some type inference issues.

Reply:
Turns out the underlying issue was that the dtype for `next_point` was int64 in one case and int32 (I think) in the other, which is fixed in https://github.com/pyviz/datashader/pull/796. With this PR alone, the performance was terrible, but that's been fixed in https://github.com/pyviz/datashader/pull/796 .